### PR TITLE
MF-471 Location Selector: Complete carbonization

### PR DESCRIPTION
**Pull request by donaldkibet**
### ticket 

[MF-471](https://issues.openmrs.org/projects/MF/issues/MF-471?filter=allissues)

### What does this PR do?

1. Complete carbonization of location selector
2. Update the loading component to use cabon inline loading icon

### Screenshoots

screencast 2021-02-23 10-07-01

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "url": "https://github.com/openmrs/openmrs-esm-login/issues"
   },
   "dependencies": {
-    "carbon-components-react": "^7.27.0",
+    "carbon-components-react": "^7.25.0",
```
Comment (FlorianRappl) [inline package.json:43]: Why this change?

**Review by FlorianRappl — COMMENTED** [review]
Looks great!

File: package.json (hunk 1 above)
Comment (donaldkibet) [reply package.json]: Its a typo

**Review by FlorianRappl — APPROVED** [review]
Thanks :beers:

File: translations/es.json [hunk 2]
```diff
@@ -3,5 +3,10 @@
   "password": "Contraseña",
   "username": "Nombre de usuario",
   "continue": "Seguir",
-  "errorMessage": "Nombre de usuario o contraseña incorrecta"
+  "errorMessage": "Nombre de usuario o contraseña incorrecta",
+  "welcome": "Bienvenido",
+  "match": "partido",
+  "selectYourLocation": "Seleccione su ubicación de la lista a continuación. Utilice la barra de búsqueda para buscar su ubicación.",
+  "searchForLocation": "Busque una ubicación.",
+  "confirm": "confirmar"
```
Comment (ujjwaltyagi355) [inline translations/es.json:11]: That looks really very good!!! Great work @donaldkibet.
